src/pages/learn.mdx

```diff
@@ -60,6 +60,7 @@ Blog posts
 - [sbt - A task engine](https://jazzy.id.au/2015/03/03/sbt-task-engine.html), 2015, James Roper
 - [sbt - A declarative DSL](https://jazzy.id.au/2015/03/04/sbt-declarative-dsl.html), 2015, James Roper
 - [testing sbt plugins](https://eed3si9n.com/testing-sbt-plugins), 2011, Eugene Yokota
+- [How I SBT](https://blog.rhetoricalmusings.com/posts/how-i-sbt1/), 2024, Vivek Raghunathan
```

Comment:
typo: Vivek Ragunathan (no h)

Reply:
@eed3si9n I saw all other entries in `learn` page use legal name to specify author. Is this a convention or is it okay to use the name `Higher-Kinded Type`?

Reply:
We should respect the chosen / preferred name of each authors. I have zero mandate for legal name.

@vivekragunathan It's up to you. We can list you as 
1. Higher-Kinded Type
2. Vivek Raghunathan (Higher-Kinded Type)
3. Higher-Kinded Type (Vivek Raghunathan)

Reply:
I like option #3.